context/app/static/js/components/entity-search/results/Results/Results.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+import ResultsTable from 'js/components/entity-search/ResultsTable';
+import Pagination from 'js/components/entity-search/results/Pagination';
+import { useStore } from 'js/components/entity-search/SearchWrapper/store';
+import ResultsTiles from 'js/components/entity-search/results/ResultsTiles/';
+import { ResultsLayout } from './style';
+
+function Results({ results }) {
+  const { view } = useStore();
+
+  return (
+    <ResultsLayout>
+      {results?.hits && (
+        <>
+          {view === 'table' && <ResultsTable hits={results.hits} />}
+          {view === 'tile' && <ResultsTiles hits={results.hits} />}
```

Comment:
```suggestion
  const ResultsComponent = {table: ResultsTable, tile: ResultsTiles}[view];

  return (
    <ResultsLayout>
      {results?.hits && (
        <>
          <ResultsComponent hits={results.hits} />
```

Some logic could be pulled out the JSX... If there were more props being repeated, might make more of a difference. Whatever you think best.